### PR TITLE
Shield Study 3:  Firefox Guide (Contextual Feature Recommendation)

### PR DESCRIPTION
**Pull request by gregglind**
Similar to #1880, static page for "Firefox Guide" (Contextual Feature Recommendation) study.

### CODE REVIEW

**Comment by gregglind** [comment]
(there is still placeholder text in there, waiting for final approval from legal and other nits, should be ready in the morning)

**Comment by EnTeQuAk** [comment]
The test failure was intermittend, I restarted the travis tests.

I guess there's an entry in `src/olympia/pages/urls.py` missing to hook up the page?

File: static/css/shield_study_3/main.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@import url("chrome://global/skin/in-content/common.css");
```
Comment (EnTeQuAk) [inline static/css/shield_study_3/main.css]: How does that work, is that page only being viewed in Firefox?

**Comment by gregglind** [comment]
1.  It will need a urls entry.  Sorry to miss that!

2.  Oh, that is true, it assumes fx only.  paging @johngruen .  I don't *expect* people to see it from outside fx.  Up to you how much faith we want to put here.

**Comment by EnTeQuAk** [comment]
> Oh, that is true, it assumes fx only. paging @johngruen . I don't expect people to see it from outside fx. Up to you how much faith we want to put here.

Well, it depends in what context the page is being shown to the user, can you elobarate on that a bit?

**Comment by gregglind** [comment]
It should only be linked to a notification box / bar.  

(Looks like I missed the window this week, eh?)

**Comment by EnTeQuAk** [comment]
Yeah, the next tag is next Tuesday so you have time till then to finalize the wording and everything so it get's deployed on Thursday.

**Comment by gregglind** [comment]
Well, I am bummed we missed this week, that sets us back a whole week.

**Comment by EnTeQuAk** [comment]
Well, this pull request still misses the relevant part in the `urls.py` as the other shield studies do and apparently the text isn't final too or is that now final? Once you add the relevant part in the urls.py we can merge this PR and you can see it on -dev and it goes online next Thursday.

**Comment by EnTeQuAk** [comment]
@gregglind reminder, if you want this to be merged and deployed on Thursday please make sure to update this PR in time, the tag will happen today around 4PM UTC

**Comment by gregglind** [comment]
Is this going or not this week?

File: static/css/shield_study_3/main.css (hunk 1 above)
Comment (gregglind) [inline static/css/shield_study_3/main.css]: That is not needed here, most of that is removed.

**Comment by johngruen** [comment]
oh hey guys, late to the party. 

*Reads* 


...yeah, using global FF styles b/c FF-only

**Comment by EnTeQuAk** [comment]
Superseeded by https://github.com/mozilla/addons-server/pull/3176

**Pull request closed without merging**